mitodl/mit-open
# Unit and detail page copy updates

### What are the relevant tickets?

Closes https://github.com/mitodl/hq/issues/4815



### Description (What does it do?)
This PR makes makes copy changes from the "units" tab in [this spreadsheet](https://docs.google.com/spreadsheets/d/1QGdAaAxnJidaI4cUG4u29wyCR5Yq494LsJe0pgRfJ5o/edit?gid=238909653#gid=238909653). Also make sure to reference notes in the ticket.

### How can this be tested?
1. checkout this branch
2. run the associated data migrations (now using the new data fixtures app) by running ```RUN_DATA_MIGRATIONS=true python manage.py migrate```
3. verify changes listed in the ticket



### Checklist:
- [ ] all changes from spreadsheet have been validated
